File: Robot/app/src/main/java/com/changren/robot/receivers/BootBroadCastReceiver.java
package com.changren.robot.receivers;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

import com.changren.robot.services.DDSService;

/**
 * Created by Admin
 * Date 2019/3/12.
 */

public class BootBroadCastReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        Log.e("BootBroadCastReceiver", "【onReceive】");
//        context.startService(SpeechService.getLaunchIntent(context));
        context.startService(DDSService.getLaunchIntent(context));
    }
}

File: Robot/appkotlin/src/main/java/com/changren/robot/receivers/BootBroadCastReceiver.kt
package com.changren.robot.receivers

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import com.changren.robot.MainActivity
import com.changren.robot.utils.XLog

/**
 * Created by Admin
 * Date 2019/4/8.
 */
class BootBroadCastReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        XLog.e("BootBroadCastReceiver", "【onReceive】")
        context.startActivity(Intent(context, MainActivity::class.java)
                .addFlags(Intent.FLAG_ACTIVITY_NEW_TASK))
    }
}